contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsSubscriptions.sol

```diff
@@ -77,9 +79,10 @@ interface IFunctionsSubscriptions {
 
   /**
    * @notice Time out all expired requests: unlocks funds and removes the ability for the request to be fulfilled
-   * @param requestIdsToTimeout - A list of request IDs to time out
+   * @param requestsToTimeoutByCommitment - A list of request commitments to time out
+   * @dev The commitment can be found on the "OracleRequest" event created when sending the request.
    */
-  function timeoutRequests(bytes32[] calldata requestIdsToTimeout) external;
+  function timeoutRequests(IFunctionsRequest.Commitment[] calldata requestsToTimeoutByCommitment) external;
```

Comment:
We will need to make the frontend team aware of this breaking change.

Reply:
Do they even plan to handle timing out for Mainnet Preview?

Reply:
If they don't plan on handling timeouts, then we are going to need to provide tooling to make it easy to do so.  This work needs to be added to the NPM package.

Reply:
Ticket here: https://smartcontract-it.atlassian.net/browse/FUN-705